sdk/eventhub/Azure.Messaging.EventHubs/tests/Azure.Messaging.EventHubs.Tests.csproj

```diff
@@ -35,4 +34,7 @@
 
   <!-- Import the Azure.Core test framework (shared source) -->
   <Import Project="..\..\..\core\Azure.Core\tests\TestFramework.props" />
+  
+  <!-- Import the references to the Azure client libraries -->
+  <Import Project="$(MSBuildThisFileDirectory)..\Azure.Client.Library.References.props" />
```

Comment:
> One last nit, and apologies for not having thought of this sooner. Could we move this file to the `samples` directory.

@jsquire 

Thank you for your nit,

Shall I update this into ```$(MSBuildThisFileDirectory)..\samples\Azure.Client.Library.References.props```

Reply:
I think I need to mea cupla here; I overlooked that this was used in both the `samples` and `tests`.    I withdraw my nit.    Your current organization is the correct one, given multiple references.   Sorry about that.

Reply:
No worries, it was a good one 👊